src/const.ts

```diff
@@ -135,11 +135,10 @@ export type ChipFamily =
   | typeof CHIP_FAMILY_ESP32H2;
 
 export const CHIP_DETECT_MAGIC_VALUES = {
-  0xfff0c101: { name: "ESP8266", family: CHIP_FAMILY_ESP8266 },
   [-999617]: { name: "ESP8266", family: CHIP_FAMILY_ESP8266 },
   0x00f01d83: { name: "ESP32", family: CHIP_FAMILY_ESP32 },
   0x000007c6: { name: "ESP32-S2", family: CHIP_FAMILY_ESP32S2 },
-  0x00000009: { name: "ESP32-S3", family: CHIP_FAMILY_ESP32S3 },
+  0x9: { name: "ESP32-S3", family: CHIP_FAMILY_ESP32S3 },
   0xeb004136: { name: "ESP32-S3(beta2)", family: CHIP_FAMILY_ESP32S3 },
```

Comment:
Doubt this one will also be matched as expected.

Reply:
> Doubt this one will also be matched as expected. ( `0xeb004136` )

@TD-er The better solution for this issue is on #28